src/Makefile

```diff
@@ -1,13 +1,19 @@
-FLAGS = -Wall
+FLAGS = -Wall -std=c99
```

Comment:
This makes gcc throw an error for using asm() in test_shvalloc (line 44). -std=gnu99 would make it happy, I don't know if we want/care.